### DISCUSSION TITLE
Error: [ lang count (3) and file count (4) don't match or file count is 0 ]

### DISCUSSION THREAD

**Discussion by ghost** [MrGreensWorkshop/MrGreen-JekyllTheme#192] [Problems] [answered]
I am trying to add one more language to the web page following the tutorial to add a language, but when I add a new language I get this error in the section where I can change the interface language on the web page:
`[ lang count (3) and file count (4) don't match or file count is 0 ].`
If more information about the error is required I will give it.

**Comment by MrGreensWorkshop** [comment, marked as answer]
Hi, Please watch [these videos](https://www.youtube.com/playlist?list=PLAymxPbYHgl-fFy5can7uZBMJtFWVcphD), if you still don't get it, check this [commit](https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/commit/7dbc13465453a6eccb169225988c782bdec3732b).
